osaca/osaca.py

```diff
@@ -484,23 +473,23 @@ def iaca_asm(self):
         code = self.srcCode
```

Comment:
`srcCode` is initialized in `store_src_code()` as a List of Strings, but used here as a String. Is this intended?

Reply:
`store_src_code()` stores `srcCode` is a String or a list of Strings depending on the given parameter `iacaFlag`, because in case of a marked kernel with IACA markers instead of high level OSACA markers, the regex is applied on `srcCode` as one string. Therefore this behavior is intended.